app/services/gov_delivery/request_builder.rb

```diff
@@ -35,10 +35,14 @@ def self.send_bulletin_xml(topic_ids, subject, body, options = {})
             xml.cdata options[:header]
           } if options[:header]
           xml.footer {
-            xml.cdata options[:footer]
-          } if options[:footer]
+            xml.cdata options[:footer] || default_footer
+          }
         }
       }.to_xml
     end
+
+    def self.default_footer
+      @default_footer ||= File.read(File.expand_path(File.join(__dir__, 'templates', 'default_footer.html')))
```

Comment:
I think you are correct a rails view might be nicer, however I left it as is for the moment as this keeps it in line with the GovUkDelivery (Python app) that has the same file. Ideally, this should be moved once we have killed GovUkDelivery.